README.md

```diff
@@ -1,330 +1,259 @@
-
-# Getting Started
-If you don't want to install anything to try Hamilton, we recommend trying [www.tryhamilton.dev](https://www.tryhamilton.dev/?utm_source=README).
-Otherwise, here's a quick getting started guide to get you up and running in less than 15 minutes.
-If you need help join our [slack](https://join.slack.com/t/hamilton-opensource/shared_invite/zt-1bjs72asx-wcUTgH7q7QX1igiQ5bbdcg) community to chat/ask Qs/etc.
-For the latest updates, follow us on [twitter](https://twitter.com/hamilton_os)!
-
-## Installation
-Requirements:
-
-* Python 3.8+
-
-To get started, first you need to install hamilton. It is published to pypi under `sf-hamilton`:
-> pip install sf-hamilton
-
-Note: to use the DAG visualization functionality, you should instead do:
-> pip install "sf-hamilton[visualization]"
-
-While it is installing we encourage you to start on the next section.
-
-Note: the content (i.e. names, function bodies) of our example code snippets are for illustrative purposes only, and don't reflect what we actually do internally.
-
-## Hamilton in <15 minutes
-Hamilton is a new paradigm when it comes to building datasets (in this case we'll use Hamilton to create columns of a
-dataframe as an example. Otherwise hamilton can handle _any_ python object.
-
-Rather than thinking about manipulating a central object (dataframe in this case),
-you instead declare the components (columns in this case)/intermediate results you want to create, and the inputs that are required. There
-is no need for you to worry about maintaining this object, meaning you do not need to think about any "glue" code;
-this is all taken care of by the Hamilton framework.
-
-For example, rather than writing the following to manipulate a central dataframe object `df`:
-```python
-df['col_c'] = df['col_a'] + df['col_b']
-```
+Hamilton supports Python 3.8+. We include the optional `visualization` dependency to display our Hamilton DAG.
 
-you would write
-```python
-def col_c(col_a: pd.Series, col_b: pd.Series) -> pd.Series:
-    """Creating column c from summing column a and column b."""
-    return col_a + col_b
+```bash
+pip install "sf-hamilton[visualization]"
 ```
-In diagram form:
-![example](hamiltondag.png)
-The Hamilton framework will then be able to build a DAG from this function definition.
 
-So let's create a "Hello World" and start using Hamilton!
+To use the Hamilton UI, install the `ui` and `sdk` dependencies.
 
-### Your first hello world.
-By now, you should have installed Hamilton, so let's write some code.
+```bash
+pip install "sf-hamilton[ui,sdk]"
+```
 
-1. Create a file `my_functions.py` and add the following functions:
-```python
-import pandas as pd
+To try Hamilton in the browser, visit [www.tryhamilton.dev](https://www.tryhamilton.dev/?utm_source=README)
 
-def avg_3wk_spend(spend: pd.Series) -> pd.Series:
-    """Rolling 3 week average spend."""
-    return spend.rolling(3).mean()
+# Key Features
```

Comment:
Capitalization is not consistent -- second word lowercase is what hte rrest of the post is